gui/src/pages/WorkspaceInfo.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import Layout from '../components/Layout.svelte';
+  import WorkspaceNav from '../components/WorkspaceNav.svelte';
+  import CenterFormPanel from '../components/form/CenterFormPanel.svelte';
+  import { api } from '../lib/api';
+  import Spinner from '../components/Spinner.svelte';
+
+  export let params = { workspace: '' };
+
+  const workspaceId = params.workspace;
+  const workspace = api.workspace(workspaceId);
+  const workspaceRoute = `/workspaces/${encodeURIComponent(workspaceId)}`;
+</script>
+
+<Layout>
+  <WorkspaceNav {workspaceId} active="Dashboard" slot="navbar" />
+  <CenterFormPanel title="Workspace details" backRoute={workspaceRoute}>
+    {#await workspace.describeSelf()}
```

Comment:
Nice @BenElgar , can I use `workspace.describeSelf()` to avoid the `describeWorkspaces().then( [find workspace with same id] )` I mention in #335 ?

Reply:
I don't see why not.